src/CMakeLists.txt

```diff
@@ -281,12 +281,13 @@ add_subdirectory(json_spirit)
 include_directories(SYSTEM "${CMAKE_SOURCE_DIR}/src/xxHash")
 include_directories(SYSTEM "${CMAKE_SOURCE_DIR}/src/rapidjson/include")
 
+find_package(fmt 5.2.1 QUIET)
+if(NOT fmt_FOUND)
+  message(STATUS "Could not find fmt, will build it")
+  add_subdirectory(fmt)
```

Comment:
we need to continue using `seastar/fmt`.

Reply:
@tchaikov Shall I modify this PR to reflect "seastar/fmt" dir ?

Reply:
@pdvian yeah, to be specific. we just need to move the
```cmake
  find_package(fmt 5.2.1 QUIET)		
   if(NOT fmt_FOUND)		
     message(STATUS "Could not find fmt, will build it")		
     add_subdirectory(seastar/fmt)		
   endif()
```
block out of `if(WITH_SEASTAR)`.